build-in-container/README.md

```diff
@@ -14,7 +14,7 @@ The mariner-docker-builder.sh script presents these options <br />
 
 Optional arguments <br />
   --mariner_dir             directory to use for Mariner artifacts (SPECS, toolkit, ..). Default is the current directory <br />
-  --RPM_repo_file           Path to a custom repo file. Please see [here](./README.md#sample-custom-repo) for sample custom repo file  <br />
+  --RPM_repo_file           Path to custom repo file(s) (must end in .repo). Please see [here](./README.md#sample-custom-repo) for sample custom repo file. Provide multiple files with comma (,) as delimiter  <br />
```

Comment:
Our tools use ` ` as our delimiter, we should probably continue that pattern here.

Reply:
Thanks for the suggestion, but are we ok with making this user-facing change? The current code allows multiple comma delimited RPM_container_URL(s)

Reply:
Actually, the multiple comma delimited RPM_container_URL(s) was broken, and is fixed in this PR. So, this might be ok to change now

Reply:
Chatted offline, going with --flag "one two three"